### PR TITLE
Removed the ssd in the option menu and provided new interface for ssd

### PR DESCRIPTION
**Pull request by nicolasseng**
Yolo und ssd sind nun im selben option menu wobei auch das interface für ssd geändert wurde.

### CODE REVIEW

**Review by FredericLala — COMMENTED** [review]
The implementation looks good, at a later point we should probably unify the looks of the UI.

But could you delete the yolov8l.pt and yolov8m.pt files, because they should already be in data/preTrainedYolo folder. Or at least they should normally be created there.
+
The yolov8_config.yaml, this file will be created individually for each user, so it should not be in the rep.

File: UI/uiFoundation.py [hunk 1]
```diff
@@ -16,15 +16,14 @@
...
-
 # --- / 
 # -- / function to select mode to display at runtime
 def SelectProgramMode()  : 
     '''
     function displaying a streamlit menu with several options
     upon selection of a module to execute, said module is executed 
     '''
-    appModeSelection = ["Readme File", "Run SSD", "Yolov8", "Show the Code"];
+    appModeSelection = ["Readme File", "Run ObejctDetection", "Show the Code"];
```
Comment (ScatteredDrifter) [inline UI/uiFoundation.py:26]: Typo here, maybe correct before merging ^

File: UI/uiRunningApp.py [hunk 2]
```diff
@@ -11,76 +14,75 @@
...
     )
     
     # --- /  styling sidebar
-    
-    input_option = st.sidebar.radio(
-        "Select input type: ", ["image", "video", "livestream"]
-    )
-    #TODO selection of possible images!
-    
+    # --- /
```
Comment (ScatteredDrifter) [inline UI/uiRunningApp.py:38]: Looks fine! Was merged from another position

File: yolo/yolov8_config.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+train: none/images
```
Comment (ScatteredDrifter) [inline yolo/yolov8_config.yaml:1]: this file should be deleted pls

File: data/.DS_Store
Comment (ScatteredDrifter) [inline data/.DS_Store:1]: delete pls, metadata not relevant for program ^

File: yolov8l.pt
Comment (ScatteredDrifter) [inline yolov8l.pt:1]: delete pls

File: yolov8m.pt
Comment (ScatteredDrifter) [inline yolov8m.pt:1]: delete pls ^

File: UI/uiYolo.py [hunk 4]
```diff
@@ -33,6 +36,16 @@ def runYoloInterface():
 
     st.title("Object Recognition Dashboard")
     st.sidebar.title("Settings")
+
+    objectDetectionSelected: Optional[str] = None
+    ObjectDetection:list = ['Yolo', 'SSD']
+    sourceTypeSelected: Optional[str] = st.sidebar.radio(
+        "Select Object Detection: ", ObjectDetection)
+    
+    if sourceTypeSelected == 'SSD':
```
Comment (ScatteredDrifter) [inline UI/uiYolo.py:45]: not the ideal solution I think, but for now It should do the job.

**Review by ScatteredDrifter — CHANGES_REQUESTED** [review]
Thanks for contributing, your works looks well mostly. 
I've added some comments regarding files that should be deleted, otherwise I'm fine with your changes.

**Review by SimonCashew — APPROVED** [review]
Alle gewünschten Änderungen vorgenommen

**Pull request merged**